src/data/api/Questionnaire.ts

```diff
@@ -3,45 +3,26 @@ import { Questionnaire } from '../../models/Questionnaire';
 import { apiEndpoints } from '../routes.json';
 import * as ApiRequest from './ApiRequest';
 import { generateFormDataByQuestion } from '../../helpers/formDataHelper';
+import { AxiosResponse } from 'axios';
 
-async function getQuestionnairesByCodes(codes: string[]): Promise<Questionnaire[]> {
-  const response = await ApiRequest.postRequest(
-    apiEndpoints.questionnaire.base_endpoint,
-    apiEndpoints.questionnaire.getQuestionnaireById,
-    {},
-    { codes: codes }
-  );
-  return response.data;
-}
-
-async function getAllQuestionnaireDataByCode(code: string): Promise<Questionnaire> {
-  const response = await ApiRequest.getRequest(
+async function getAllQuestionnaireDataByCode(code: string): Promise<AxiosResponse> {
+  return await ApiRequest.getRequest(
     apiEndpoints.questionnaire.base_endpoint,
     apiEndpoints.questionnaire.getAllQuestionnaireDataByCode,
     { code: code }
   );
-  return response.data;
 }
 
 async function saveQuestionnaireByCode(code: string, questionnaire: Questionnaire) {
+  const formData = new FormData();
+  formData.append('questionnaire', questionnaire);
```

Comment:
Waarom wordt die meegegeven in de formdata? Is toch gewoon een json?

Reply:
Ik denk dat dit gefixt moet worden in de volgende sprint. Want daar staat een user story voor

Reply:
Is goed